specs/experimental/l2_blob_tx.md

```diff
@@ -0,0 +1,43 @@
+# L2 Blob Transaction
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [L2 Blob Transaction](#l2-blob-transaction)
+  - [Motivation](#motivation)
+  - [How It Works](#how-it-works)
+  - [Reference Implementation](#reference-implementation)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+The proposal aims to integrate BLOB transaction support into the OP Stack. BLOB transactions are gaining popularity on Layer 1 (L1), with applications including BLOB inscriptions, Rollups (as an optional feature), and EthStorage. However, given the high costs associated with L1, it is crucial to implement BLOB transaction support in the OP Stack. This implementation would significantly reduce migration costs for projects and users looking to leverage BLOB transactions in a more cost-effective environment.
+
```

Comment:
## Overview of Implementation

### Derivation
Depending on L2 BLOB DA configuration:
- Sequencer-self-hosted: Skip actual L2 BLOB data during derivation
- DAC: Verify the DAC signatures during derivation
- L1 DA: Decode and verify L2 BLOB from L1 BLOB during derivation
- DA challenge (Plasma): Check L1 challenge contract to see if a BLOB is challenged during derivation

### BLOB Gas Cost
What is the formula of L2 BLOB gas cost?

### BLOB P2P Network
Compared to L1 BLOB processing in P2P network, what we have done?

### Other Minor Changes
...